commands/command.md

```diff
@@ -95,34 +122,39 @@ Command flags is @array-reply containing one or more status replies:
    4) (integer) 1
    5) (integer) 1
    6) (integer) 1
+   ...
 ```
 
-Some Redis commands have no predetermined key locations.  For those commands,
-flag `movablekeys` is added to the command flags @array-reply.  Your Redis
+Some Redis commands have no predetermined key locations. For those commands,
+flag `movablekeys` is added to the command flags @array-reply. Your Redis
 Cluster client needs to parse commands marked `movablekeys` to locate all relevant key positions.
 
-Complete list of commands currently requiring key location parsing:
+Partial list of commands currently requiring key location parsing:
```

Comment:
```suggestion
Here are a few examples of commands that are marked with `movablekeys`:
```

Reply:
i now realize that he was trying to document a full list, because for each of them he provided the instructions of how to find the keys if you write your own parser.
so maybe we do need a complete list.
@itamarhaber WDYT? ("complete" lists are bound to get outdated and incomplete)

Reply:
@itamarhaber ^